UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Webjob.UnitTests/Services/FakeFileHelper.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections.Generic;
+
+using UKHO.ExchangeSetService.Common.Helpers;
+using UKHO.ExchangeSetService.Common.Models.FileShareService.Response;
+
+namespace UKHO.ExchangeSetService.Webjob.UnitTests.Services
+{
+    public class FakeFileHelper : IFileSystemHelper
+    {
+        public bool CheckAndCreateFolderIsCalled = false;
+        public bool CreateFileContentWithBytesIsCalled = false;
+
+        public void CheckAndCreateFolder(string folderPath)
+        {
+            CheckAndCreateFolderIsCalled = true;
+        }
+
+        public void CreateFileContentWithBytes(string outputFileName, byte[] content)
+        {
+            CreateFileContentWithBytesIsCalled = true;
+        }
+
+        public bool CheckDirectoryAndFileExists(string rootPath, string zipFilePath)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public bool CheckFileExists(string filePath)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public bool CreateFileContent(string fileName, string content)
+        {
+            throw new System.NotImplementedException();
+        }        
+
+        public void CreateZipFile(string rootPath, string zipFileName)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public CustomFileInfo GetFileInfo(string filePath)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public List<FileDetail> UploadCommitBatch(BatchCommitMetaData batchCommitMetaData)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public byte[] UploadFileBlockMetaData(UploadBlockMetaData UploadBlockMetaData)
+        {
+            throw new System.NotImplementedException();
+        }
+    }
+}
```

Comment:
I think this would normally be referred to as a spyFileHelper but otherwise implemented as I would have expected.

It may be worth renaming this in due course after we merge to main as this pattern may need adopting elsewhere and we could end up with more spy's!